performance.lic

```diff
@@ -0,0 +1,156 @@
+custom_require.call(%w(common common-travel))
+  
+class Performance
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    no_pause_all
+    startTime = Time.new.to_i
+
+    @settings = get_settings
+    @song_list = get_data('perform').perform_options
+    UserVars.song = @song_list.first.first unless UserVars.song
+    fput('release ecry') if DRStats.bard?
```

Comment:
❓ Why release ECRY? Is it so the performance buff will not cause you to play a more-difficult-than-normal song?

Reply:
❌ Since you're relying on `DRStats`, please add `drinfomon` to the `custom_require` list

Reply:
That's right for releasing ECRY.  Without this you can end up in situations where the script can't accurately play a song, since you might be playing considerably different songs if ECRY is up vs when its not.  Releasing ECRY also extends the range of training with zills by about 150 ranks

Reply:
Is the `require` call necessary or just convention?  The script works without it

Reply:
ECRY: Cool, thanks for the explanation